src/linkedList.ts

```diff
@@ -1,7 +1,9 @@
+import { conjoinedMsg } from './messageObjects';
+
 class linkedListNode
 {
 
-    public data: any;
+    public data: conjoinedMsg;
```

Comment:
LinkedList is a general library and not necessarily tied to this project - let's make sure to remove references to twitchToDiscord so that we can use this in other projects 👍

Reply:
It's either we make the data related towards the project, or have type casting on each time we access a linkedListNode. It makes no since on why we don't just add the conjoinedMsg type to our data, due to the fact it will never change from that type in this project.

Up to you though, I personally dislike the look of this with the type casting.
Simply makes life in Typescript easier. Either way, with type casting it looks ugly.
```ts
const fetchedNode: linkedListNode = ...;

(fetchedNode.data as conjoinedMsg).message.reply(...);
```

Reply:
Made it a generic type, just out of spite. If you don't like it LMK. It works.